src/services/EventBus.ts

```diff
@@ -21,6 +21,8 @@ import mitt from 'mitt'
 export type Events = {
 	[key: EventType]: unknown
 	'audio-player-ended': number
+	'chat-relay-message-received': { token: string, message: ChatMessage }
+	'chat-relay-supported': void
```

Comment:
Maybe rename it to be in line with other signaling messages emits? (e.g. users-joined)
```suggestion
	'signaling-message-received': { token: string, message: ChatMessage }
```